packages/documentation-v7/.storybook/preview.ts

```diff
@@ -47,10 +45,9 @@ const preview: Preview = {
       container: DocsLayout,
       source: {
         excludeDecorators: true,
+        transform: (snippet: string) => format(snippet, prettierOptions),
```

Comment:
There was this comment "ensure the string is not empty ('') because the Source component breaks if it is" in the last version. Isn't it still necessary? I left it in my PR: https://github.com/swisspost/design-system/pull/1638/files but I was not sure.

Reply:
I checked and this is no longer the case: returning an empty string makes the Source component to display the original snippet.